src/js/views/clinicians/shared/clinicians_views.js

```diff
@@ -16,6 +16,12 @@ const i18n = intl.clinicians.shared.clinicianViews;
 
 const GroupsComponent = GroupsManagerComponent.extend({
   removeMemberGroup(group) {
+    if (this.member.isNew()) {
```

Comment:
Previously this component wasn't available to a new clinician, but we don't want the warning below in that situation